tofu/geom/_comp.py

```diff
@@ -46,6 +46,18 @@
     'volume': 'abs',
 }
 
+
+def _check_float(var=None, varname=None, vardef=None):
+    if var is None:
```

Comment:
It's the name of the variable, this is a generic abstract function to check that a variable is of type float or int and raise an automated error message otherwise

Reply:
and the value to a default value if None